images/extension-builders/rust/commands.sh

```diff
@@ -53,15 +54,11 @@ help:  make sure Cargo workspace includes a library crate with name '${lib_name}
 "
 	fi
 
-	local destination_file="$1"
+	local destination_file="${GETENVOY_WORKSPACE_DIR}/$1"
 	if [[ -n "${destination_file}" ]]; then
 		log_message "     Copying *.wasm file to '${destination_file}'"
-
-		destination_file="${GETENVOY_WORKSPACE_DIR}/${destination_file}"
-		local tmp_file="${destination_file}.tmp"
-		mkdir -p "$(dirname "${tmp_file}")"
-		cp "${cargo_output_file}" "${tmp_file}"
```

Comment:
if two-step copy had a reason to exist, there are no comments explaining why. I'm removing this, but it can be added back with a comment why regardless of if I'm around or not.